utils/grpc/grpc_client.go

```diff
@@ -0,0 +1,31 @@
+package grpcutils
```

Comment:
bit: best to make the folder name match the package name

Reply:
https://github.com/onflow/flow-go/pull/1224/commits/ad89c53ad5fb896550eb38d9c3ee8f1dd802b6dd